### PR TITLE
task 611 : Mutual

### PR DESCRIPTION
**Pull request by amirrezamirzaei**
task 611 using [Mutual](https://arxiv.org/abs/2004.04494)

### CODE REVIEW

File: tasks/README.md [hunk 1]
```diff
@@ -443,4 +443,5 @@ Name | Summary | Category
 `task607_sbic_intentional_offense_binary_classification` | Determine whether the post is intentionally offensive or not | Binary Classification
 `task608_sbic_sexual_offense_binary_classification` | Determine whether the post is sexually offensive/explicit or not | Binary Classification
 `task609_sbic_potentially_offense_binary_classification` | Determine whether the post is potentially offensive or not | Binary Classification
-`task610_conllpp_ner` | Recognize and label proper nouns (Named Entity Recognition) | NER/Labeling  
\ No newline at end of file
+`task610_conllpp_ner` | Recognize and label proper nouns (Named Entity Recognition) | NER/Labeling
+task611_mutual_multi_turn_dialogue |
```
Comment (yeganehkordi) [inline tasks/README.md]: I guess you have forgotten to complete the README file.

**Review by yeganehkordi — COMMENTED** [review]
Thanks! It looks good to me.
Some minor comments:
- Please replace `continue . your` with `continue. Your`, correct the typo of the "comma", and capitalize the first word of the sentences in the definition and the explanation of the negative example.
- I think the category of this task should be "MCQ". (Take a look at the comments here: #243 )

**Comment by danyaljj** [comment]
Thanks! Looks good to me! Leaving it here for @swarooprm to take a look before merging.

**Review by swarooprm — APPROVED** [review]
LGTM!
I was thinking of these new tasks from the Mutual dataset, see if it makes sense for the data you have.

- Given a conversation (without mentioning roles), identify the speaker (binary classification problem)
- Given a conversation up to m turns, and nth (n>m) dialogue, identify the speaker 
- Given a conversation up to m turns, and nth dialogue, determine if n>m or n<m
- Similar to the task you have, instead of asking to predict the next dialogue, predict the first dialogue or missing dialogue of a conversation.
etc.

**Pull request merged**